server/vulnerabilities/cve.go

```diff
@@ -69,6 +69,26 @@ var (
 	theClock clock.Clock = clock.C
 )
 
+func getNVDCVEFeedFiles(vulnPath string) ([]string, error) {
+	var files []string
+	err := filepath.Walk(vulnPath, func(path string, info os.FileInfo, err error) error {
```

Comment:
Consider using `WalkDir` here.

Reply:
Ah yes, I believe `WalkDir` is supposed to be faster too.